quiche/src/lib.rs

```diff
@@ -1350,6 +1355,20 @@ impl Config {
     pub fn set_disable_dcid_reuse(&mut self, v: bool) {
         self.disable_dcid_reuse = v;
     }
+
+    /// Configures tracking unknown transport params.
+    ///
+    /// To enable, pass `Some` `usize` with the maximum
+    /// amount of space to expend to track the ids and
+    /// values of unknown transport parameters. To disable,
+    /// pass `None`.
```

Comment:
See comment below on the signature but it probably wants to be tweaked to something like 
```suggestion
    /// Enables tracking unknown transport parameters. 
    /// The provided `size` is the maximum size in bytes that
    /// will be allocated to storing transport parameter IDs
    /// and their values.
```